dss-demo-webapp/src/main/resources/logback.xml

```diff
@@ -1,8 +1,11 @@
 <configuration>
 
 	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-		<encoder>
-      		<pattern>%d %5p | %t | %-55logger{55} | %m %n</pattern>
+		 <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="ch.qos.logback.contrib.json.classic.JsonLayout">
+                <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSSX</timestampFormat>
+				<jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter"/>
+            </layout>
```

Comment:
Does it store the logs to a separate file or prints them to console? If I remember well, the requirement was to store the logs to a file. I see that "ch.qos.logback.core.ConsoleAppender" is used.

Reply:
There is no need to store logs on the file system. This is being run in a containerized environment with ephemeral storage - we are feeding those log to our centralized logging system (data dog) where logs are stored and managed. It is actually preferable NOT to store logs on the file system as it can cause disks to fill up

Reply:
Ok, I misunderstood the request then. You may also change the appender to "ch.qos.logback.core.ConsoleAppender" in my branch, to make logs reported within a console.